### ISSUE TITLE
Upgrade PHP version

### ISSUE THREAD

**Issue by benoitcdmnt** [bitnami/bitnami-docker-drupal#134]
**Description**

Hello, is there a way to upgrade the PHP version in this image ? If yes, what is the best proper way of doing it ?

[The support for 7.3 will end soon](https://www.php.net/supported-versions.php) and I need to move to 7.4

[The 7.3 version is no more recommended too by drupal](https://www.drupal.org/docs/system-requirements/php-requirements#php_required) 

Thank you

**Comment by marcosbc** [comment]
Hi @benoitcdmnt, thanks for reporting this! We expect to update the PHP version for all of our containers before its EOL.

However, in the case of Drupal, we are going to priorize them changes for today and ensure we get a container image released soon afterwards, as it's clear that it's not the version that is recommended anymore.

In the meantime, you can apply these changes to the container's Dockerfile in order to build it on your side:

```diff
@@ -8,19 +8,19 @@

 COPY prebuildfs /
 # Install required system packages and dependencies
-RUN install_packages acl ca-certificates curl dirmngr gnupg gzip libargon2-1 libaudit1 libbsd0 libbz2-1.0 libc6 libcap-ng0 libcom-err2 libcurl4 libexpat1 libffi6 libfftw3-double3 libfontconfig1 libfreetype6 libgcc1 libgcrypt20 libglib2.0-0 libgmp10 libgnutls30 libgomp1 libgpg-error0 libgssapi-krb5-2 libhogweed4 libicu63 libidn2-0 libjemalloc2 libjpeg62-turbo libk5crypto3 libkeyutils1 libkrb5-3 libkrb5support0 liblcms2-2 libldap-2.4-2 liblqr-1-0 libltdl7 liblzma5 libmagickcore-6.q16-6 libmagickwand-6.q16-6 libmcrypt4 libmemcached11 libmemcachedutil2 libncurses6 libnettle6 libnghttp2-14 libp11-kit0 libpam0g libpcre3 libpng16-16 libpq5 libpsl5 libreadline7 librtmp1 libsasl2-2 libsodium23 libsqlite3-0 libssh2-1 libssl1.1 libstdc++6 libsybdb5 libtasn1-6 libtidy5deb1 libtinfo6 libunistring2 libuuid1 libwebp6 libx11-6 libxau6 libxcb1 libxdmcp6 libxext6 libxml2 libxslt1.1 libzip4 procps tar unzip zlib1g
-RUN . /opt/bitnami/scripts/libcomponent.sh && component_unpack "php" "7.3.29-6" --checksum 7daac9464051ab7ae075ec20e0d24ee99e95e133d2bca9e3bf78c1e6a3994424
+RUN install_packages acl ca-certificates curl dirmngr gnupg gzip libaudit1 libbsd0 libbz2-1.0 libc6 libcap-ng0 libcom-err2 libcurl4 libexpat1 libffi6 libfftw3-double3 libfontconfig1 libfreetype6 libgcc1 libgcrypt20 libglib2.0-0 libgmp10 libgnutls30 libgomp1 libgpg-error0 libgssapi-krb5-2 libhogweed4 libicu63 libidn2-0 libjemalloc2 libjpeg62-turbo libk5crypto3 libkeyutils1 libkrb5-3 libkrb5support0 liblcms2-2 libldap-2.4-2 liblqr-1-0 libltdl7 liblzma5 libmagickcore-6.q16-6 libmagickwand-6.q16-6 libmcrypt4 libmemcached11 libmemcachedutil2 libncurses6 libnettle6 libnghttp2-14 libonig5 libp11-kit0 libpam0g libpcre3 libpng16-16 libpq5 libpsl5 libreadline7 librtmp1 libsasl2-2 libsodium23 libsqlite3-0 libssh2-1 libssl1.1 libstdc++6 libsybdb5 libtasn1-6 libtidy5deb1 libtinfo6 libunistring2 libuuid1 libwebp6 libx11-6 libxau6 libxcb1 libxdmcp6 libxext6 libxml2 libxslt1.1 libzip4 procps tar unzip zlib1g
+RUN . /opt/bitnami/scripts/libcomponent.sh && component_unpack "php" "7.4.22-1" --checksum 3ef2eb24c358a84f03bf1eee1747a1379bac7ab2c54601abae4423a72ed6f843
 RUN . /opt/bitnami/scripts/libcomponent.sh && component_unpack "apache" "2.4.48-1" --checksum 0296bc7dd5f621eba6f948433a6c7fc04a45876f42f6395cea5888b707f1ad11
 RUN . /opt/bitnami/scripts/libcomponent.sh && component_unpack "mysql-client" "10.3.31-0" --checksum 92b99c64a3ffe10fd94ef9ecd94d2dd65a172404c726093eb50b3b293523559d
-RUN . /opt/bitnami/scripts/libcomponent.sh && component_unpack "libphp" "7.3.29-1" --checksum bb5e5663ed26e35eac840f2f98d5beb0a458bad653dceb6efba0d8fd4918d673
+RUN . /opt/bitnami/scripts/libcomponent.sh && component_unpack "libphp" "7.4.22-1" --checksum 02c944de0d39725adebf8a16960ceff5448f9d75d8ea8dc8228692baadee7659
 RUN . /opt/bitnami/scripts/libcomponent.sh && component_unpack "render-template" "1.0.0-3" --checksum 8179ad1371c9a7d897fe3b1bf53bbe763f94edafef19acad2498dd48b3674efe
 RUN . /opt/bitnami/scripts/libcomponent.sh && component_unpack "gosu" "1.14.0-0" --checksum 3e6fc37ca073b10a73a804d39c2f0c028947a1a596382a4f8ebe43dfbaa3a25e
 RUN . /opt/bitnami/scripts/libcomponent.sh && component_unpack "drupal" "9.2.4-1" --checksum 820441b34477a5e6ea5a176c5804c516a2d47d52500e0d229b5f32288514caab
```

**Comment by benoitcdmnt** [comment]
Great thank you. I was able to test and switch to php 8 + php-fpm too with some image modification.

**Comment by marcosbc** [comment]
Hi @benoitcdmnt, just wanted to let you know that we just released tags `8.9.18-debian-10-r1` and `9.2.4-debian-10-r2` of the `bitnami/drupal` container image, which include PHP 7.4.
